pkg/agent/option.go

```diff
@@ -59,6 +59,9 @@ type ClusterRegistrationOptions struct {
 	// PredictorAddress specifies the address of predictor
 	PredictorAddress string
 
+	// donot collect metrics from metric-server by default
```

Comment:
UseMetricsServer specifies whether to collect metrics from metrics server

Reply:
ok